config.json

```diff
@@ -64,7 +64,8 @@
     "bowling",
     "dot-dsl",
     "custom-set",
-    "forth"
+    "forth",
+    "pangram"
```

Comment:
Putting this at the end for now since reordering will be done after adding `topics` and `difficulties` for all exercises.